package.json

```diff
@@ -16,7 +16,7 @@
     "Alexandre Chau"
   ],
   "dependencies": {
-    "@graasp/sdk": "1.7.0",
+    "@graasp/sdk": "github:graasp/graasp-sdk#285-add-a-utility-type-to-make-some-properties-of-an-object-optional",
```

Comment:
As @pyphilia already mentioned, isn't it better to use a new version of the SDK from NPM if the feature is available in a higher version?